ppr-ui/README.md

```diff
@@ -1,8 +1,7 @@
 # ppr-ui
 
 ## Project setup
-```
-npm install
+```npm install
```

Comment:
We don't normally include style changes in a PR that is a functional change.
It tends to bury the changes amongst other stylistic changes, and those have already made it through a PR cycle.